src/rebar_utils.erl

```diff
@@ -158,11 +158,21 @@ sh(Command0, Options0) ->
     ?DEBUG("Port Cmd: ~s\nPort Opts: ~p\n", [Command, PortSettings]),
     Port = open_port({spawn, Command}, PortSettings),
 
-    case sh_loop(Port, OutputHandler, []) of
-        {ok, _Output} = Ok ->
-            Ok;
-        {error, {_Rc, _Output}=Err} ->
-            ErrorHandler(Command, Err)
+    case os:type() of
+        {win32, _} ->
```

Comment:
So again, why do you need this? Isn't it enough to expect that `rebar_utils:sh("robocopy blah blah"...)` may return something different from `{ok, _}`?